moped-editor/src/views/projects/projectView/ProjectFunding/ProjectFundingTable.js

```diff
@@ -445,7 +445,7 @@ const ProjectFundingTable = ({ handleSnackbar }) => {
 
   // saves row update, either editing an existing row or saving a new row
   const processRowUpdate = (updatedRow, originalRow) => {
-    const updateProjectFundingData = updatedRow;
+    const updateProjectFundingData = { ...updatedRow };
```

Comment:
@mddilley I did this to address the bug you found -- now `updatedRow` isn't being touched so if the row hasn't changed, then when the Promise resolves, it keeps the same data

Reply:
awesome! thank you for tracking this down and fixing! I retested and the bug is gone. 🙏 🙌